settings.gradle

```diff
@@ -3,4 +3,8 @@ include ':sample-library-model'
 include ':sample'
 include ':sample-app'
 include ':test-java-lib'
-include ':compile-bench'
\ No newline at end of file
+include ':test-java-lib-jarinfer'
```

Comment:
Maybe not for this PR, but it'd be nice to not have all the modules at the top-level and to have the jarinfer modules in their own folder

Reply:
@subarnob Can you take a look at how hard it would be to do this for this PR. If it's simple enough, I would love to have all these targets under a `jarinfer/` top level dir on the repo from day one :) (if it's hard to do, I can take a look later)